pkgs/games/katago/default.nix

```diff
@@ -17,7 +17,8 @@
 , gpuEnabled ? true
 , useAVX2 ? false
 , cudaSupport ? false
-, useTcmalloc ? true}:
+, useTcmalloc ? true
+, bigBoardEnabled ? false}:
```

Comment:
Sorry I didn't notice before... but these options are usually named `enableSomething`. Can you rename this and `gpuEnabled` as well?

Reply:
Easier done than said.

Reply:
Thank you.